examples/lambda/variables.tf

```diff
@@ -6,9 +6,10 @@ variable "name" {
   type        = string
 }
 
-variable "ecr_image_uri" {
-  description = "The private ECR image URI for consul-lambda-registrator."
+variable "lambda_registrator_image" {
```

Comment:
This looks like a breaking change to me. We should probably mark it as a breaking change in the changelog.

Reply:
I'm also assuming this breaks the acceptance tests https://github.com/hashicorp/terraform-aws-consul-lambda/blob/db2b43aba3ef3cd9dd5de9b8da3a6eae5b4d6653/test/acceptance/tests/setup/main.tf#L69

Reply:
I agree. I missed this in my review. We need to open a PR to update the CHANGELOG for this.